tomcat8/hooks/run

```diff
@@ -4,13 +4,18 @@ exec 2>&1
 
 echo "Starting Apache Tomcat"
 
-if path={{pkgPathFor "core/jdk8"}}; then
+if path=$(hab pkg path core/jdk8); then
```

Comment:
This is causing Travis to complain. Can you explain why the change from `pkgPathFor`?

Reply:
This fixes issue #2033 because `hab pkg path` does not require the referenced java pkg to be in the tomcat `pkg_deps`. Where if it was `pkgPathFor` it would. By changing this tomcat can be loaded as a service so long as a version of java is installed without being wrapped in a plan.

Reply:
I have thought about this one a bit and I'm at a loss how to best fix this. If we use `pkgPathFor` then we need to add a version of java to the `pkg_deps`. In my opinion though adding a version of java is not a good idea since that is more a requirement of the application using tomcat. Is the recommendation then to add better instruction for how to use tomcat in the readme?